### ISSUE TITLE
Creating a project board

### ISSUE THREAD

**Issue by siddhant-khisty** [afro-coder/trekking-your-way-to-linux#1]
Continuing from the talk we had on [LinkedIn](https://www.linkedin.com/posts/leon-nunes_trekking-your-way-to-linuxdevops-showwcase-activity-7080533566001954816-Ow5K?utm_source=share&utm_medium=member_desktop), here are a few topics we could start with as a v0.1 version. We can make individual issues for all the listed topics, and track them through the project boards. Please let me know what you think, and any additional content we can add.

- [ ] Setting up a VM for using Linux on Windows
- [ ] Navigation
- [ ] File & directory management
- [ ] User Management
- [ ] Shell Operators
- [ ] Different Shells and why are there different Shells
- [ ] Shell vs Terminal (Terminal Emulator)
- [ ] Systemd Units - Start, disable, enable
- [ ] Services vs Daemons
- [ ] Cron
- [ ] Linux Audio - ALSA + pipe wire to start with
- [ ] Vim/NVim/LVim
- [ ] Desktop Environments vs Display Manager (Login Manager)
- [ ] Bootloaders
- [ ] Linux directory structure
- [ ] Basic Networking
- [ ] Network Manager
- [ ] Bluetooth

As a reference for creating the .md files for all topics, let's refer to the almighty [Arch Wiki](https://wiki.archlinux.org/title/Table_of_contents)

**Comment by afro-coder** [comment]
Hey @siddhant-khisty thank you this looks amazing! we should probably create different `mountains` depending on what folks wanna learn for example, a system-admin mountain, and a linux-desktop, we can then split titles into those and link them as needed!

**Comment by siddhant-khisty** [comment]
Sounds good. I'll update the above message into these two `mountains` tomorrow. Let's have these two ones for the time being, and we can add more later on.

However, some topics such as navigation would come under every single mountain. How should we proceed with those? Should we simply add them under a `Fundamentals` category?

**Comment by afro-coder** [comment]
Yeah, I also created an issue in the project board for Mountain Names, something like `Trek Fundamentals` or `Trekking Basics` should do the trick

**Comment by siddhant-khisty** [comment]
@afro-coder The project boards are empty for me. I think you might have created a project in your account. You might need to link it here

**Comment by afro-coder** [comment]
It was private, made it public

**Comment by baibhavKumar1** [comment]
What's a mountain in this context?

**Comment by afro-coder** [comment]
Anything to which you'd like to trek too, in short, a goal, could be hills too, where you stop and pick a path, but that's too far fetched at this moment.

**Comment by AniketNS** [comment]
I just came here after watching the @siddhant-khisty Linux 101 video. I want to share what I've learnt from that video. 

Should I create a new issue for that or where should I write that. Can you give me a short roadmap?

**Comment by afro-coder** [comment]
Hey @AniketNS sure, this repo hasn't been active since I didn't get a chance to go ahead with things, the goal was to build goal based approaches.

Let me see if I can get a sample out soon
